openam-authentication/openam-auth-ldap/src/main/resources/amAuthLDAP_ja.properties

```diff
@@ -28,6 +28,7 @@
 # Portions Copyrighted 2012-2015 ForgeRock AS.
 # Portions Copyrighted 2014 Open Source Solution Technology Corporation
 # Portions Copyrighted 2014 Nomura Research Institute, Ltd
+# Portions Copyrighted 2019 Open Source Solution Technology Corporation
```

Comment:
`2019` -> `2014-2019`.
And please delete previous line.

Reply:
IC.